### PR TITLE
setup: remove ``resolved_records`` directory

### PR DESCRIPTION
**Pull request by drjova**
Signed-off-by: Harris Tzovanakis <me@drjova.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,7 +21,7 @@
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 # JSON files, generated from YAML
 inspire_schemas/records/**/*.json
-inspire_schemas/resolved_records/**/*.json
+inspire_schemas/unresolved_records/**/*.json
```
Comment (michamos) [inline .gitignore]: nice, but you still added the yaml to the commit

Comment (drjova) [reply .gitignore]: 🤕

Comment (michamos) [reply .gitignore]: still there :sweat_smile:

File: Pipfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+[[source]]
```
Comment (michamos) [inline Pipfile]: also this shouldn't be committed.

**Pull request closed without merging**